src/LogicApp/LogicApp/Cmdlets/LogicApp/GetAzureLogicAppRunHistoryCommand.cs

```diff
@@ -55,8 +65,17 @@ public override void ExecuteCmdlet()
             base.ExecuteCmdlet();
             if (string.IsNullOrEmpty(this.RunName))
             {
-                var enumerator = LogicAppClient.GetWorkflowRuns(this.ResourceGroupName, this.Name).GetEnumerator();
-                this.WriteObject(enumerator.ToIEnumerable<WorkflowRun>(), true);
+                Page<WorkflowRun> page = LogicAppClient.GetWorkflowRuns(this.ResourceGroupName, this.Name);
```

Comment:
Please preserve the use of 'var' where possible like here

Reply:
Do keep the new variable name however as it is more descriptive of the content of the variable.